static/js/gear_menu.js

```diff
@@ -41,6 +41,23 @@ exports.initialize = function () {
         }
     });
 
+    $('#contact_support').on('click', function (e) {
+        new ClipboardJS('#contact_support', {
+            text: function () {
+                var data = page_params.zulip_administrator;
+                return data;
+            },
+        });
+        e.stopPropagation();
+        $(this).html('<i class="fa fa-address-book" aria-hidden="true"></i> Email copied');
```

Comment:
Our convention is generally to use `e.target` instead of `this` for click handlers.  At the top of the function you can say something like `var menu_item = $(e.target)`, then the rest of your code will be more clear:

~~~
menu_item.html(...);
menu.item.addClass(...)
setTimeout(function () {
    menu_item.html(...)
    menu_item.removeClass(...)
}